hashicorp/waypoint
# ui: Add Docker image/tag information and Overview section

#1520
## Original UI
<img width="955" alt="Screen Shot 2021-09-30 at 5 24 20 PM" src="https://user-images.githubusercontent.com/60155296/135532216-f75355f6-0a4b-45cf-ae01-d62114a4a77e.png">

## Updated UI
<img width="952" alt="Screen Shot 2021-09-30 at 5 24 02 PM" src="https://user-images.githubusercontent.com/60155296/135532265-43bc7bb4-ef52-49a9-9bfe-cb3f6c80c9bf.png">

## How to Test
1. Pull this branch down `ui/docker-version`
2. Run a local Waypoint server ([see here](https://github.com/hashicorp/waypoint/tree/main/ui))
3. Create a project
4. `waypoint up`
5. Check the deployment details page
